src/Casbin.php

```diff
@@ -116,15 +116,15 @@ public function __construct(?Config $config = null)
 
         $this->config = $config;
 
-        if (!$config->getAdapter() instanceof Adapter) {
-            $this->adapter = new DatabaseAdapter();
-        }
-
-        if ($logger = $this->config->getLoggerClass() ?? 'EasySwoole\Permission\Logger') {
-            if (class_exists($logger)) {
-                $logger = $logger::getInstance();
```

Comment:
It is wrong to call the `getInstance` method in this place. The log processor set by the user is not necessarily a singleton.